lxl-web/src/routes/(app)/[[lang=lang]]/[fnurgel=fnurgel]/Holdings.svelte

```diff
@@ -40,10 +40,19 @@
 	function missingAtLeastOneLinkToItem() {
 		return bibIds.find((id) => !hasLinksToItem(id.bibId) && !hasLoanReserveLink(id.bibId));
 	}
+
+	function hasOpeningHoursEtc() {
```

Comment:
I noticed this is not really working. The function seems to return true while the info is still missing. See for example "Bilblioteken i Lomma"
http://localhost:5173/cv4j8tk59bwd3p88?holdings=Print

This is because the opening hours are arrays that has multiple empty elements. You'll have to join it before checking, i.e

```
		if (id) {
			const openingHours = linksByBibId[id.bibId]?.[holder.sigel]?.[BibDb.OpeningHours];
			const adress = linksByBibId[id.bibId]?.[holder.sigel]?.[BibDb.Address];

			return ((openingHours || adress?.join('').trim()))
		} return false;
```

Reply:
Sorry, it's adress that's often empty strings. The function would return true for all three of these.

<img width="687" alt="Skärmavbild 2025-06-03 kl  14 49 28" src="https://github.com/user-attachments/assets/c60794cd-a41f-43e5-b3ce-a4c43f5665fc" />